src/test/kotlin/org/rust/lang/core/completion/RsKeywordCompletionContributorTest.kt

```diff
@@ -589,6 +589,30 @@ class RsKeywordCompletionContributorTest : RsCompletionTestBase() {
         "fn foo<T /*caret*/>() {}"
     """)
 
+    fun `test const inside trait`() = checkCompletion("const", """
+        pub trait Bar {
+            /*caret*/
+        }
+    ""","""
+        pub trait Bar {
+            const /*caret*/
+        }
+    """)
+
+    fun `test const inside impl`() = checkCompletion("const", """
```

Comment:
Could you please add tests for other keywords (not only for `const`)?

Reply:
`org.rust.lang.core.completion.RsKeywordCompletionContributorTest#checkCompletion(java.util.List<java.lang.String>, java.lang.String, java.lang.String)` can be usefull for that